.github/workflows/bundleTest-thor.md

```diff
@@ -0,0 +1,353 @@
+      sudo chown -R $USER:$USER /opt/HPCCSystems
+      sudo xmlstarlet ed -L -u 'Environment/Software/ThorCluster/@slavesPerNode' -v 2 -u 'Environment/Software/ThorCluster/@channelsPerSlave' -v 1 /etc/HPCCSystems/environment.xml
+  ```
+- **Install ML Dependencies**  
+Install the necessary Machine learning library dependecies
+  ```yaml
+  - name: Install ML Dependencies
+      shell: "bash"
+      run: |  
+        sudo apt install libsvm-dev libsvm-tools 
+        sudo pip install tensorflow numpy keras   
+  ```
+- **Start HPCC-Platform**  
+Setup the core generation and start the HPCC Platform
+  ```yaml
+  - name: Start HPCC-Platform
+          shell: "bash"
+          run: |
+            export LANG="en_US.UTF-8"
+            ulimit -c 100
+            echo 'core_%e.%p' | sudo tee /proc/sys/kernel/core_pattern
+            sudo update-locale
+            sudo /etc/init.d/hpcc-init start
+  ```
+- **Core generation test**  
+This step is used to check if the core generation is working as expected. This is an optional step. This step runs only when it is specified in the inputs via workflow call.
+  ```yaml
+  - name: Core generation test
+          if: ${{ inputs.test-core-file-generation }}
+          shell: "bash"
+          run: |
+              echo """
+              boolean seg() := beginc++ #option action
+              #include <csignal>
+              #body
+              raise(SIGABRT);
+              return false;
+              endc++;
+              output(seg()); """ > crash.ecl
+              
+              ecl run -t hthor crash.ecl
+          continue-on-error: true
+  ```
+- **Get test from Github**  
+Install the ECL bundles specified in ` BUNDLES_TO_TEST:` from github.
+
+  ```yaml
+  - name: Get test from Github
+          shell: "bash"
+          run: |
+            IFS=' ' read -a BUNDLES_TO_TEST <<< $BUNDLES_TO_TEST
+            BUNDLES_COUNT=${#BUNDLES_TO_TEST[@]}
+            for ((i=0; i<$BUNDLES_COUNT; i++))
+            do
+                BUNDLE_NAME=${BUNDLES_TO_TEST[i]}
+                BUNDLE_REPO="https://github.com/hpcc-systems/${BUNDLES_TO_TEST[i]}.git"
+                INSTALL_CMD="ecl bundle install --update --force ${BUNDLE_REPO}"
+            
+                tryCountMax=5
+                tryCount=$tryCountMax
+                tryDelay=1m
+
+                while true
+                do
+                    cRes=$( ${INSTALL_CMD} 2>&1 )
+                    retCode=$?
+                    if [[ $retCode -ne 0 ]]
+                    then
+                        tryCount=$(( $tryCount-1 ))
+
+                        if [[ $tryCount -ne 0 ]]
+                        then
+                            sleep ${tryDelay}
+                            continue
+                        else
+                            break;
+                        fi
+                    else
+                        BUNDLE_VERSION=$( echo "${cRes}" | egrep "^$BUNDLE_NAME" | awk '{ print $2 }' )
```

Comment:
This BUNDLE_VERSION is acquired, but not used. 
In the original version of bundleTest.sh some information has been logged like bundle name, repo name, install CMD and the finally installed bundle version, or error message if the install failed.
I think they are useful information and would be nice to put them into the action code as well in this documentation.